tests/unit/style/properties/serialization.rs

```diff
@@ -905,6 +905,17 @@ mod shorthand_serialization {
                 repeat-x padding-box subtract;"
             );
         }
+
+        #[test]
+        fn serialize_mask_position_with_multiple_values() {
+            let block_text = "\
+                mask-position: 1px 2px, 4px 3px;";
```

Comment:
I based this test on the second one (with background-position), but agree, it will look better in one line ;)